scoresheets/CleanTable.tex

```diff
@@ -2,18 +2,17 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[5]{60}{Picking tableware and cutlery for transportation to the dishwasher}
-	\scoreitem[5]{180}{Placing the tableware and cutlery inside the dishwasher}
+	\scoreitem[5]{240}{Placing the tableware and cutlery inside the dishwasher}
 
 	\scoreheading{Bonus Rewards}
 	\scoreitem{200}{Opening the dishwasher door}
 	\scoreitem{100}{Pulling out the dishwasher rack}
-	\scoreitem[5]{40}{Placing an item correctly in the dishwasher}
-	\scoreitem{300}{Placing the dishwasher tab inside the dishwasher}
+	\scoreitem{200}{Placing all items correctly in the dishwasher}
```

Comment:
While some objects may be placed correctly by accident I think this makes it too easy for things to accidentaly go wrong. If one objects placement goes wrong you lose out on 200 points. I would try to avoid these high risk high reward situations. This may discourage teams from even trying to go for a correct placement.

Reply:
We should remember this is a bonus, as we don't give partial points for "trying" to open the dishwasher's door or the rack, we should not give partial points for "trying" to classify objects (or, in this case, give free points even if a team is not trying to do the classification itself as it wouldn't make any sense to give points to a team if the dishwasher's door opens by accident).